code/game/machinery/constructable_frame.dm

```diff
@@ -181,6 +181,7 @@
 				if(component_check)
 					P.play_tool_sound(src)
 					var/obj/machinery/new_machine = new circuit.build_path(loc)
+					new_machine.setDir(dir) //NSV13 - set machine's dir to frame's dir

```

Comment:
You are aware that this might brake normal SS13 machines if they are really old right?

Reply:
It is unlikely for them to be constructed but we will need to be aware of this.

Reply:
Changed to add a keyword argument to New instead